content/02.introduction.md

```diff
@@ -1,29 +1,25 @@
 ## Introduction
+
+Scientific communication has been around since the beginning of science where the main form of communication was through scientific meetings [@raw:smith2006trouble].
+Communicating through scientific journals started back in the 17th century [@raw:smith2006trouble] but didn't pick up until the 1960s where National Institutes of Health (NIH) created an exchange network that allowed scientists to share unreferred manuscripts with one another [@doi:10.1371/journal.pbio.2003995].
+These networks were brough to a halt due to pressure from scientific journals; however, this sparked a movement to begin using preprints within the scientific community [@doi:10.1371/journal.pbio.2003995]. 

```

Comment:
I thought this article made the point that the movement was shut down. How does this relate to the change today, which appears to be more inspired out of physics?